SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -26,8 +26,12 @@ void ULayeredLBStrategy::Init()
 	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
 	const ASpatialWorldSettings* WorldSettings = GetWorld() ? Cast<ASpatialWorldSettings>(GetWorld()->GetWorldSettings()) : nullptr;
 
-	if (SpatialGDKSettings->bOverrideLoadBalancing || WorldSettings == nullptr || !WorldSettings->bEnableMultiWorker)
+	bool bIsMultiWorkerDisabled = (SpatialGDKSettings->bOverrideMultiWorker.IsSet() && !SpatialGDKSettings->bOverrideMultiWorker)
+		|| (!SpatialGDKSettings->bOverrideMultiWorker.IsSet() && (WorldSettings == nullptr || !WorldSettings->bEnableMultiWorker));
```

Comment:
Can this be const? Also on the second line, why do you need `!SpatialGDKSettings->bOverrideMultiWorker.IsSet()`? If `(WorldSettings == nullptr || !WorldSettings->bEnableMultiWorker)` is satisfied then I assume we want to disable?

Reply:
If the override is set we never want to consider the world settings.
If the override is true, always try use the world lb layers.
If the override is false, always use the default lb strategy. 
If the override is not set, then use the world setting bEnableMultiWorker.

We do indeed want it disabled.  

It will make it const.